### ISSUE TITLE
Open in Editor should separate spec and config

### ISSUE THREAD

**Issue by kanitw** [vega/vega-embed#337]
Currently if we provide a spec and a _separate_ config object to Vega-embed, and then click open in editor.  The editor would have no config, but the config would instead get merged into the spec.

I think we should instead avoid merging them, but show the provided config in the config tab.

**Comment by domoritz** [comment]
Good idea. The editor currently doesn't support getting the config separately via the post API.

**Comment by kanitw** [comment]
Hmm is that true? 

https://github.com/vega/vega-embed/blob/master/src/embed.ts#L427 

It seems like we never merge the config with spec and instead post the config to the editor.

**Comment by domoritz** [comment]
Oh right. Then we only need to fix the editor.

It should be easy to modify the line here: https://github.com/vega/editor/blob/master/src/components/app.tsx#L39.

Let's transfer this issue to the editor.
